pkgmgr/registry.go

```diff
@@ -91,6 +91,37 @@ docker run --rm -ti ghcr.io/blinklabs-io/mithril-client:0.7.0-1 $@
 			},
 		},
 	},
+	{
+		Name:        "ogmios",
+		Version:     "v5.6.0-mainnet",
+		Description: "Ogmios, a WebSocket & HTTP server for Cardano, providing a bridge between Cardano nodes and clients.",
+		InstallSteps: []PackageInstallStep{
+			{
+				Docker: &PackageInstallStepDocker{
+					ContainerName: "ogmios",
+					Image:         "cardanosolutions/ogmios:v5.6.0-mainnet",
+					Env: map[string]string{
```

Comment:
Ogmios doesn't read env, but if we wanted to use some and then pipe them through to the command arguments, that could work.